crates/figma_import/src/transform_flexbox.rs

```diff
@@ -1268,7 +1269,7 @@ fn visit_node(
                         font_size,
                         font_family: sub_style.font_family.clone(),
                         font_weight,
-                        font_style: font_style, // Italic or Normal
+                        font_style: FontStyle::Normal, //sub_style.font_style.unwrap_or(style.font_style),
```

Comment:
The "Italic or Normal" comment can be added back.

Reply:
I made a change to this line in [#1345 ](https://github.com/google/automotive-design-compose/pull/1345/files#diff-ebf114a94b7727e76848f9d545ffb662a15fec0c0db77a2c88ade5d79268a1c7L1242) , I believe it now should be 
`font_style: font_style, // Italic or Normal`

Reply:
Fixed.